test/LaunchDarkly.EventSource.Tests/Stubs/StubHandler.cs

```diff
@@ -57,140 +58,141 @@ public HttpRequestMessage AwaitRequest()
         }
     }
 
-    public abstract class StubResponse

```

Comment:
Most of this is just linefeed changes. I've tried to normalize all the linefeeds with `.gitattributes` but there were some files that hadn't been updated.